stellargraph/core/graph.py

```diff
@@ -906,18 +926,30 @@ def str_node_type(count, nt):
             sep="\n  ",
         )
 
-        # FIXME: it would be better for the schema to just include the counts directly
-        unique_ets = self._unique_type_triples(return_counts=True)
```

Comment:
I think we can remove the `return_counts` parameter from `_unique_type_triples` now, because it's now unused after this?